test_runner/README.md

```diff
@@ -3,21 +3,13 @@
 This directory contains integration tests.
 
 Prerequisites:
-- Python 3.7 or later
-    - Development headers may also be needed to build `psycopg2` from source.
```

Comment:
I've removed this line in c5b8fd71703e90ff1bed8cfad803515e74ac84ff because we use `psycopg2-binary` instead of `psycopg2` since #631.

Although I'm a little concerned it says "The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources." [here](https://pypi.org/project/psycopg2-binary/).

Reply:
We use it only for testing purposes, so I don't see any problems with that